main/webapp/modules/core/scripts/dialogs/expression-preview-dialog.js

```diff
@@ -187,7 +187,11 @@ ExpressionPreviewDialog.Widget.prototype._renderHelpTab = function() {
 
 ExpressionPreviewDialog.Widget.prototype._renderHelp = function(data) {
     var elmt = this._elmts.expressionPreviewHelpTabBody.empty().width(this._tabContentWidth);
-    
+    $("<a />", {
+      href: "https://docs.openrefine.org/manual/grelfunctions",
+      text: "View the GREL reference in the OpenRefine User Manual for further details",
```

Comment:
I think this text should come from a translation file to support multiple languages

Reply:
yeah done ,this just slipped off my mind